src/globus_cli/commands/collection/list.py

```diff
@@ -0,0 +1,109 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command
+from globus_cli.principal_resolver import default_identity_id_resolver
+from globus_cli.services.gcs import get_gcs_client
+from globus_cli.termio import FORMAT_TEXT_TABLE, formatted_print
+
+STANDARD_FIELDS = [
+    ("ID", "id"),
+    ("Display Name", "display_name"),
+    ("Owner", default_identity_id_resolver.field),
+    ("Collection Type", "collection_type"),
+    ("Storage Gateway ID", "storage_gateway_id"),
+]
+
+
+class ChoiceSlugified(click.Choice):
+    """
+    Allow either hyphens or underscores, e.g. both 'mapped-collections' or
+    'mapped_collections'
+    """
+
+    def convert(self, value, param, ctx):
+        if value is None:
+            return None
+        return super().convert(value, param, ctx).replace("_", "-")
```

Comment:
Looks like this is only doing one of the two replacements. If you want to accept `mapped_collections` as an input, match against `mapped-collections` (as show in helptext), and get `mapped_collections` as an output, I think you need
```python
super().convert(value.replace("_", "-"), param, ctx).replace("-", "_")
```

Reply:
The expanded tests made a similar complaint (yay).